src/types/todoData.ts

```diff
@@ -0,0 +1,10 @@
+export default interface TodoData {
+	isLoading: boolean;
+	data: Todos[];
+}
+
+export interface Todos {
```

Comment:
The interface/class name should be singular and clear. You can rename it as Task instead.